netket/tools/_cpu_info.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import platform
+import importlib
+
+PLATFORM = sys.platform
+
+
+def available_cpus():
```

Comment:
Maybe put this directly into `netket.utils`, as it could have a use outside the `info` scripts in `tools`?

Reply:
I don't know...
I don't think this will work forever. 
If XLA people change things, and you will use current netket with future jax, you might get wrong things printed (without ways of knowing, because JAx does not really surface this information in any useable way). Also there are flags that affect parallelism, so this is a guarantee of nothing.

For a debug utility I think it's ok.
But I would like users not to write code relying on this (because it's just an upper bound that works today), so I am against the inclusion in netket itself.

Reply:
Good point.